### PR TITLE
[TGER-47] add and register feedback policy

### LINKED ISSUE #21: Add Policies

### PR DESCRIPTION
**Pull request by devjk1**
closes #21

### CODE REVIEW

**Comment by joshtorres** [comment]
Good job. We still need to bring over the permissions in a migration and policy tests for this specific package. You can use this as an example: https://github.com/tipoff/discounts/pull/15/files

File: src/Policies/FeedbackPolicy.php [hunk 1]
```diff
@@ -0,0 +1,96 @@
...
+use Tipoff\Support\Contracts\Models\UserInterface;
+
+class FeedbackPolicy
+{
+    use HandlesAuthorization;
+
+    /**
+     * Determine whether the user can view any models.
+     *
+     * @param  \App\Models\User  $user
```
Comment (drewroberts) [inline src/Policies/FeedbackPolicy.php]: The @param for psalm needs to be updated from `\App\Models\User` to `Tipoff\Support\Contracts\Models\UserInterface`

File: src/Policies/FeedbackPolicy.php [hunk 2]
```diff
@@ -0,0 +1,96 @@
...
+    public function viewAny(UserInterface $user): bool
+    {
+        return $user->hasPermissionTo('view feedbacks') ? true : false;
+    }
+
+    /**
+     * Determine whether the user can view the model.
+     *
+     * @param  \App\Models\User  $user
+     * @param  \App\Feedback  $feedback
```
Comment (drewroberts) [inline src/Policies/FeedbackPolicy.php]: The @param for Feedback also needs to be updated here

File: src/Policies/FeedbackPolicy.php (hunk 1 above)
Comment (joshtorres) [reply src/Policies/FeedbackPolicy.php]: @devjk1 let me know I this works after you test it again.

**Comment by drewroberts** [comment]
> Good job. We still need to bring over the permissions in a migration and policy tests for this specific package. You can use this as an example: https://github.com/tipoff/discounts/pull/15/files

Let's do that in another PR so I can merge another PR that updates the package service provider.

**Review by drewroberts — APPROVED** [review]
Thank you! 🍵

**Pull request merged**